adapter/service/parameters/specification.go

```diff
@@ -0,0 +1,85 @@
+package parameters
+
+import (
+	"fmt"
+
+	"github.com/futurehomeno/fimpgo"
+	"github.com/futurehomeno/fimpgo/fimptype"
+
+	"github.com/futurehomeno/cliffhanger/router"
+)
+
+// Specification creates a service specification.
+func Specification(
+	resourceName,
+	resourceAddress,
+	address string,
+	groups []string,
+	parameterSizes []int,
```

Comment:
we don't implement parameterSize in Cliff implementation

Reply:
Good to know, fixed.